scripts/benchmark_queue.py

```diff
@@ -0,0 +1,116 @@
+'''
+A script that benchmarks the queue performance, can be used to compare the performance 
+of the queue on a given branch vs the main branch. By default, runs 100 jobs in batches
+of 20 and prints the average time per job. The inference time for each job (without the
+network overhead of sending/receiving the data) is 0.5 seconds. Each job sends one of:
+a text, image, audio, or video input and the output is the same as the input.
+
+Navigate to the scripts/ directory and run:
+>> python benchmark_queue.py
+
+You can specify the number of jobs to run and the batch size with the -n parameter:
+>> python benchmark_queue.py -n 1000
+
+The results are printed to the console, but you can specify a path to save the results 
+to with the -o parameter:
+>> python benchmark_queue.py -n 1000 -o results.json
```

Comment:
nit: I had to run this from the top level of the repo, otherwise it complained about not being able to find the image files.

Reply:
Ah thanks, will fix so it's consistent with the instructions